seedDB/README.md

```diff
@@ -1,17 +1,20 @@
-These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+# cBioPortal Seed Database
 
-- DB schema version **1.3.0**
+These files are MySQL database dump files for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+
+- DB schema version **2.1.0**
 
 Which is suitable for:
 
-- release version **1.3.1** of cBioPortal.  
-- release version **> 1.3.1** of cBioPortal (:warning: with possible migration script step). 
+- release version **1.5.0** of cBioPortal.  
```

Comment:
we can state **1.5.1** here. I just checked, and 1.5.1 does *not* introduce updates to DB schema.